src/js/education-wizard.js

```diff
@@ -0,0 +1,65 @@
+{
+  function toggleClass(element, className) {
+    element.classList.toggle(className);
+  }
+  function openState(element) {
+    element.setAttribute('data-state', 'open');
+  }
+  function closeState(element) {
+    element.setAttribute('data-state', 'closed');
+  }
+  function isParent(element) {
+    const isParentElement = !element.dataset.selectedForm;
+    return isParentElement;
+  }
+  function closeStateAndCheckChild(alternateQuestion, container) {
+    closeState(alternateQuestion);
+    if (!isParent(alternateQuestion)) {
+      return;
+    }
+    const optionsElements = alternateQuestion.querySelectorAll('input');
+    const options = Array.prototype.slice.call(optionsElements);
+    options.forEach(option => {
+      if (option.dataset.nextQuestion) {
+        const alternateChild = option.dataset.nextQuestion;
+        const alternateChildElement = container.querySelector(`[data-question=${alternateChild}]`);
+        return closeStateAndCheckChild(alternateChildElement, container);
+      }
+      return false;
+    });
+  }
+  function reInitWidget() {
+    const container = document.querySelector('.expander-container');
+    const button = container.querySelector('.expander-button');
+    const content = container.querySelector('.expander-content');
+    const radios = container.querySelectorAll('input');
+    const apply = container.querySelector('#apply-now-button');
+    radios.forEach(radio => {
+      radio.addEventListener('click', () => {
```

Comment:
This should probably be a change event. With click, this is going to run unnecessarily if someone clicks on an already selected option.